### ISSUE TITLE
Jaws doesn't announce count of element in radiogroup in IE11

### ISSUE THREAD

**Issue by yugu0114** [FreedomScientific/VFO-standards-support#449]
### Summary

> The first test case in this list is failed in IE11.
> https://a11ysupport.io/tests/tech__aria__aria-required-radiogroup#assertion-aria-radiogroup_role-convey_name--jaws-ie
> Also it doesn't work in the example:
> 1. https://a11ysupport.io/tests/html/aria/aria-required-radiogroup.html
> 2. Navigate to 'Cat'

### Expected result

> '1 of 3' is announced

### Actual result

> '1 of 3' is not announced

### Example

> https://a11ysupport.io/tests/html/aria/aria-required-radiogroup.html

#### JAWS version and build number
2020.2008.24 ILM

 
#### Operating System and version
Win 10

#### Browser and version:
IE v. 11.1082.18362.0 / update v. 11.0.210 (KB4577010)

**Comment by stevefaulkner** [comment]
@yugu0114 It is not announced because the information is not exposed in the accessibility tree. IE accessibility support is limited to MSAA it does not support IA2. In modern browsers such as Firefox the `setsize` and `posinset` properties are exposed as IA2 object attributes and JAWS will use it to announce position and set size. Note from my testing NVDA does not announce this information in IE 11 either. The [ARIA core accessibility API mapping spec](https://www.w3.org/TR/core-aam-1.1/#mapping_state-property_table) is useful for understanding the origin of such properties.
